<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Передаётся n сообщений по каналу связи. Каждое сообщение с вероятностью 
p независимо от других искажается. С.в. η - число искажённых сообщений</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Кол-во экспериментов:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="experiments">
          <property name="text">
           <string>100</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>n:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="n">
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>p:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="p">
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculate">
          <property name="text">
           <string>расчет</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Ввод границ интервалов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intervals">
        <property name="text">
         <string>3.5 7.5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>alpha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="alpha">
        <property name="text">
         <string>0.7</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QTableWidget" name="table">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>101</height>
           </size>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>y_i</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>n_i</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>n_i/n</string>
           </property>
          </row>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Fη(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="F"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
        <item>
         <widget class="QTableWidget" name="table2">
          <attribute name="horizontalHeaderVisible">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Eη</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>x¯</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>|Eη - x¯|</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dη</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>S^2</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>|Dη - S^2|</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Me^</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>R^</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>F¯η(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="F_sample"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>max|Fη(x) -F¯η(x)| =</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="D">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="table3">
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>y_j</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>P({η= y_j})</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>η_j / n</string>
           </property>
          </row>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>max|η_j/n - P({η = y_j})| = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="max_freq_prob">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="S">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="N">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>F(R0) =</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="hyp">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="Fres">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
